components/mgmtworker/scripts/configure.py

```diff
@@ -75,5 +71,18 @@ def configure_logging():
     utils.move(config_file_temp_destination, config_file_destination)
 
 
+def prepare_snapshot_permissions():
+    pgpass_location = '/root/.pgpass'
+    destination = join(HOMEDIR, '.pgpass')
+    utils.chmod('400', pgpass_location)
+    utils.chown(MGMTWORKER_USER, MGMTWORKER_GROUP, pgpass_location)
+    utils.sudo(['mv', pgpass_location, destination])
+    utils.sudo(['chgrp', MGMTWORKER_GROUP, '/opt/manager'])
```

Comment:
This folder specifically- it's where it creates a snapshot_credentials (IIRC) folder. I'm not sure why it creates it, but it gets grumpy if it can't.